cmd/dbc/docs.go

```diff
@@ -0,0 +1,149 @@
+// Copyright 2025 Columnar Technologies Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/cli/browser"
+	"github.com/columnar-tech/dbc"
+)
+
+var dbcDocsUrl = "https://docs.columnar.tech/dbc/"
+
+// Support drivers without a docs URL defined in the index
+var fallbackDriverDocsUrl = map[string]string{
+	"duckdb":     "https://arrow.apache.org/adbc/current/driver/duckdb.html",
```

Comment:
I wonder if we should link to `https://duckdb.org/docs/stable/clients/adbc` instead?

Reply:
Hrm, yeah we should. And we should update the DuckDB page in the ADBC docs a bit too (I'll do that).